framework/src/context_vulkan.cpp

```diff
@@ -872,156 +872,53 @@ namespace gvk
 		return extent;
 	}
 
+	// TODO it makes more sense for swap chain creation to be inside window class perhaps?	
```

Comment:
Yes, I agree. It used to be in `context_vulkan` because queue selection might depend on the existence of an already created swap chain. But I see that you've already moved some code into `window`. That's fine.